microbenchmarks/pyaes.py

```diff
@@ -0,0 +1,508 @@
+"""Simple AES cipher implementation in pure Python following PEP-272 API
```

Comment:
Maybe we could put this file in minibenchmarks?  My thinking is that microbenchmarks can be for testing single language features (ex: how fast can we throw exceptions, or do function calls, etc), and that minibenchmarks can be for testing small programs.